app/scripts/geocoder/geocoder-service.js

```diff
@@ -18,7 +18,7 @@
         var searchOutFields = 'StAddr,City,Postal';
         var searchCategories = [
             'Street Address',
```

Comment:
Not sure what the difference is between street address and postal, but until I added postal, no addresses on Spring Garden would show up in CAC, for example.